packages/contracts-bedrock/contracts/L1/ResourceMetering.sol

```diff
@@ -5,6 +5,7 @@ import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable
 import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
 import { Burn } from "../libraries/Burn.sol";
 import { Arithmetic } from "../libraries/Arithmetic.sol";
+import { SystemConfig } from "../L1/SystemConfig.sol";
```

Comment:
`ResourceConfig` struct should be defined within `ResourceMetering`, not `SystemConfig`. Better relationship between the two contracts.